### PR TITLE
Feature/us1468 package and fingerprinting

### PR DESCRIPTION
**Pull request by silbermm**

### CODE REVIEW

File: crossroads.net/app/index.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en" ng-app="crossroads">
+
+<head>
+  <base href="/"></base>
```
Comment (charlie-retzler) [inline crossroads.net/app/index.html]: I think this was modified in the HTML5mode PR

File: crossroads.net/app/index.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  <script type="text/ng-template" id="on-blur-messages">
+      <span ng-message="minlength"><span ng-bind-html="$root.MESSAGES.invalidPassword.content | html"></span></span>
+      <span ng-message="unique"><span ng-bind-html="$root.MESSAGES.emailInUse.content | html"></span></span>
+      <span ng-message="email"><span ng-bind-html="$root.MESSAGES.invalidEmail.content | html"></span></span>
+      <span ng-message="invalidRouting"><span ng-bind-html="$root.MESSAGES.invalidRoutingTransit.content | html"></span></span>
+      <span ng-message="invalidAccount"><span ng-bind-html="$root.MESSAGES.invalidAccountNumber.content | html"></span></span>
+      <span ng-message="naturalNumber"><span ng-bind-html="$root.MESSAGES.invalidDonationAmount.content | html"></span></span>
+      <span ng-message="invalidZip"><span ng-bind-html="$root.MESSAGES.invalidZip.content | html"></span></span>
+  </script>
+  <ng-include src="'assets/cr.svg'"></ng-include>
```
Comment (charlie-retzler) [inline crossroads.net/app/index.html]: I think this was modified in the HTML5mode PR

File: crossroads.net/app/index.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+<body ng-controller="coreController as app" ng-class="app.asideState.position">
+  <div growl class="crossroads-growl">
+  </div>
+  <div ui-view> </div>
+</body>
+<script src="https://js.stripe.com/v2/"></script>
+<script type="text/javascript" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-5391d6a6145291c4" async="async"></script>
+<!-- build:js -->
+<script src="assets/main.js"></script>
```
Comment (charlie-retzler) [inline crossroads.net/app/index.html]: I think this was modified in the HTML5mode PR

File: crossroads.net/lib/select.js [hunk 4]
```diff
@@ -0,0 +1,1816 @@
+/*!
+ * ui-select
+ * http://github.com/angular-ui/ui-select
+ * Version: 0.12.1 - 2015-07-28T03:50:59.076Z
+ * License: MIT
+ */
+
+
```
Comment (andycanterbury) [inline crossroads.net/lib/select.js]: Does this need to be in the repo? looks like a library we should just be pulling from elsewhere.

Comment (silbermm) [inline crossroads.net/lib/select.js]: ui-select is not yet available from npm :cry:

File: crossroads.net/webpack-assets.json [hunk 5]
```diff
@@ -0,0 +1 @@
+{"main":{"js":"/assets/main.d3cbb5dea24951d273d4.js","css":"/assets/main.d3cbb5dea24951d273d4.css"}}
```
Comment (IngageBrewerM) [inline crossroads.net/webpack-assets.json]: does this file need to be checked into the repo, or just .gitignored?

**Pull request merged**